src/analytics/lambdas/custom-resource/get-source-prefix.ts

```diff
@@ -47,14 +48,18 @@ async function _handler(event: CdkCustomResourceEvent) {
   const odsEventPrefix = (event.ResourceProperties as PropertiesType).odsEventPrefix;
 
   let prefix = odsEventPrefix;
+  const projectId = (event.ResourceProperties as PropertiesType).projectId;
 
   if (odsEventPrefix.endsWith('/ods_events/') || odsEventPrefix.endsWith('/ods_events')) {
     prefix = odsEventPrefix.replace(new RegExp('/ods_events[/]?'), '');
+  } else if (odsEventPrefix.endsWith(`clickstream/${projectId}/data/ods/`)) {
+    // control plane set the prefix to clickstream/<projectId>/data/ods/
+    // the actual prefix should be clickstream/<projectId>/data/ods/<projectId>/
```

Comment:
Does the spark ETL add the additional prefix `projectId`?

Reply:
ETL saved the files to <prefix>/<projectId>/ods_events/

I supposed that control plane set the prefix ` <prefix>/<projectId>/ods_events/`, but actually, it set the prefix to `<prefix>/`, this bug was reported by @dengmingtong  this morning. 

the <prefix> is 'clickstream/<projectId>/data/ods/'